### PR TITLE
Revise the doc site with recent features

### PR DESCRIPTION
**Pull request by resouer**
Re-write some doc for v1.0 release.

### CODE REVIEW

File: docs/en/helm/component.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ so we provide a chart which only includes minimal dependencies KubeVela relies o
 
 Install the minimal flux2 chart provided by KubeVela:
 ```shell
-helm install --create-namespace -n flux-system helm-flux http://oam.dev/catalog/helm-flux2-0.1.0.tgz
+$ helm install --create-namespace -n flux-system helm-flux http://oam.dev/catalog/helm-flux2-0.1.0.tgz
```
Comment (wonderflow) [inline docs/en/helm/component.md:14]: without the `$` , the user can copy-paste  directly

**Review by wonderflow — APPROVED** [review]
Cool

File: docs/en/helm/component.md (hunk 1 above)
Comment (resouer) [reply docs/en/helm/component.md:14]: Then we should have consistency, for now all other docs have $

Comment (wonderflow) [reply docs/en/helm/component.md:14]: Some of the commands along with output should keep `$`

**Pull request merged**